clients/common/nm-meta-setting-desc.c

```diff
@@ -7252,6 +7252,31 @@ static const NMMetaPropertyInfo *const property_infos_WIRELESS[] = {
 			.typ_flags =                NM_META_PROPERTY_TYP_FLAG_ENUM_GET_PARSABLE_TEXT,
 		),
 	),
+	PROPERTY_INFO_WITH_DESC (NM_SETTING_WIRELESS_WAKE_ON_WLAN,
+		.property_type =                &_pt_gobject_enum,
+		.property_typ_data = DEFINE_PROPERTY_TYP_DATA (
+			PROPERTY_TYP_DATA_SUBTYPE (gobject_enum,
+				.get_gtype =            nm_setting_wireless_wake_on_wlan_get_type,
+				.value_infos =          ENUM_VALUE_INFOS (
+					{
+						.value = NM_SETTING_WIRELESS_WAKE_ON_WLAN_NONE,
+						.nick = "none",
```

Comment:
note, that `value_infos` is only used during set, that is, when parsing the string value. Unlike the `value_infos_get` field which is used by the getter. With set-only specifying `"none"` is redundant to what we get from the GEnumClass -- it would make sense with the getter, to ensure `"none"` has highest priority.